src/purify.js

```diff
@@ -412,8 +412,8 @@ function createDOMPurify(window = getGlobal()) {
     PARSER_MEDIA_TYPE =
       // eslint-disable-next-line unicorn/prefer-includes
       SUPPORTED_PARSER_MEDIA_TYPES.indexOf(cfg.PARSER_MEDIA_TYPE) === -1
-        ? (PARSER_MEDIA_TYPE = DEFAULT_PARSER_MEDIA_TYPE)
-        : (PARSER_MEDIA_TYPE = cfg.PARSER_MEDIA_TYPE);
```

Comment:
I thought it was unnecessary because I was already assigning it to `PARSER_MEDIA_TYPE`.